examples/VLCB_4in4out/VLCB_4in4out.ino

```diff
@@ -160,6 +161,7 @@ void setupVLCB()
   
   // register our VLCB event handler, to receive event messages of learned events
   ecService.setEventHandler(eventhandler);
+  epService.setRequestEventHandler(eventhandler);
```

Comment:
I guess you should register requestHandler here.

Reply:
Oh, no, I see what you mean.  I decided that requestHandler as a function was unnecessary as it could be done by adding the opcodes to the eventhandler making things a bit more compact